.github/config.yml

```diff
@@ -15,10 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/Layer5-celebration.png)
-
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/meshery-celebration-light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/meshery-celebration-dark.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
+  </picture>
```

Comment:
Thanks for your contribution @Ghat0tkach 😄 Everything looks good, just a little change is needed. The alt here needs to be "Meshery Welcome Image" and the srcset needs to be either empty (```""```) or any of the above two image srcset, since this is a fallback image for the browsers that do not support the dynamic rendering.

Reply:
Thanks for the feedback @anshgoyalevil , did the required changes